airgun/entities/subscription.py

```diff
@@ -107,6 +107,18 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def filter_columns(self, columns=None):
+        """Filters column headers
+        :param columns: name of column header to filter by
```

Comment:
That suggests we can pass in list or tuple, which isn't really true. We need to pass in dict that contains column labels as keys and boolean values as values. Maybe `dict with column label: boolean value pairs`?

Reply:
@mirzal 
I'll just update the comment of the param to reflect that it's a dict mapping column name to boolean value if that's alright.